### ISSUE TITLE
Allow to configure default pageSize

### ISSUE THREAD

**Issue by mahnunchik** [Graylog2/graylog2-server#15387]
## What?

Allow to configure default pageSize.

## Why?

It would be helpful to configure default pageSize for streams page: `/streams?page=1&pageSize=50`.

There is 10 by default now.

## Your Environment

* Graylog Version: Graylog 5.0.6+51f2df8 
* Elasticsearch Version: opensearchproject/opensearch:2 
* MongoDB Version: mongo:6
* Operating System: Docker/Ubuntu 22.04
* Browser version: any

**Comment by T100D** [comment]
Actually the same for the Alert pages, default should be higher then ten or to be configurable.

**Comment by mahnunchik** [comment]
I prefer to be able configure this option. For me is suitable 25 items per page.

**Comment by linuspahl** [comment]
@mahnunchik Thank you.

I am closing this issue because we recently worked on improving the streams overview. Part of it is persisting the per page information https://github.com/Graylog2/graylog2-server/issues/14293. Currently we've implemented these improvements for the streams, dashboards, saved searches, event definitions and event notifications overview and plan to implement them for further entity lists in the future.

We also have an issue about configuring a custom page size: https://github.com/Graylog2/graylog2-server/issues/14968
